packages-next/admin-ui/src/templates/index.ts

```diff
@@ -37,4 +38,5 @@ export const writeAdminFiles = (
     ({ adminUILabels: { path }, key }) =>
       ({ mode: 'write', src: itemTemplate(key), outputPath: `pages/${path}/[id].js` } as const)
   ),
+  { mode: 'write', src: apiTemplate, outputPath: 'pages/api/graphql.js' },
```

Comment:
Should be conditional based on the option.

Reply:
Yah I tried this but typescript was very angry at me. Will try again.

Reply:
The answer to TypeScript complaining will probably be to write `as const` after `mode: 'write'`